litmus-portal/frontend/src/pages/MonitoringDashboardPage/styles.ts

```diff
@@ -111,6 +121,211 @@ const useStyles = makeStyles((theme) => ({
   modal: {
     padding: theme.spacing(15, 0),
   },
+
+  selectDate: {
+    display: 'flex',
+    height: '2.9rem',
+    minWidth: '9rem',
+    marginLeft: theme.spacing(3.75),
+    border: `0.1px solid ${theme.palette.border.main}`,
+    background: theme.palette.background.paper,
+    borderRadius: 4,
+    textTransform: 'none',
+  },
+
+  displayDate: {
+    width: '100%',
+    color: theme.palette.text.primary,
+  },
+
+  rangeSelectorIcon: {
+    width: '0.625rem',
+    height: '0.625rem',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(-1),
+  },
+
+  rangeSelectorClockIcon: {
+    width: '0.825rem',
+    height: '0.825rem',
+    marginLeft: theme.spacing(-1),
+    marginRight: theme.spacing(1),
+  },
+
+  rangeSelectorPopover: {
+    marginTop: theme.spacing(37.5),
+  },
+
+  formControl: {
+    width: '9rem',
+    marginLeft: theme.spacing(1.5),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(3.75),
+    },
+    '& .MuiSelect-outlined': {
+      padding: '0.925rem',
+      '&:focus': {
+        borderColor: theme.palette.primary.main,
+      },
+      '& .MuiInputLabel-root': {
+        color: `${theme.palette.text.hint} !important`,
+        marginTop: `${theme.spacing(2)} !important`,
+      },
+    },
+  },
+
+  inputLabel: {
+    color: theme.palette.text.hint,
+    marginTop: theme.spacing(-1),
+    '&.MuiInputLabel-shrink': {
+      marginTop: theme.spacing(1),
+    },
+  },
+
+  menuListItem: {
+    color: theme.palette.text.hint,
+    height: '2.4rem',
+  },
+
+  menuListItemSelected: {
+    height: '2.4rem',
+    background: `${theme.palette.primary.light} !important`,
+    color: theme.palette.secondary.contrastText,
+    '&:hover': {
+      background: `${theme.palette.primary.light} !important`,
+    },
+  },
+
+  refreshIcon: {
+    marginRight: theme.spacing(1),
+  },
+
+  refreshText: {
+    marginTop: theme.spacing(-2.85),
```

Comment:
margin in negative should be avoided. Please restructure the divss

Reply:
will take a look for this one.

Reply:
I have checked this one, its required here as we are using MUI and it is pushing the text out of the form control area due to theme settings in litmus, it's the same case with all form controls which have been fixed this very same way.

Reply:
can you attach a screenshot of the component you are describing?

Reply:
can you attach a screenshot of the component you are describing?